config/locales/en.yml

```
@@ -1137,7 +1129,7 @@ en:
     routing_page_caption: Question %{question_number}’s routes
     routing_page_delete: Delete question %{question_number}’s route
     routing_page_edit: Edit route %{route_number}
-    routing_page_new: Add route %{route_number}
+    routing_page_new: 'Add route %{route_number}: Select an answer and where to skip to'
```

```suggestion
    routing_page_new: 'Add route %{route_number}: select an answer and where to skip to'
```
Just need to change from capital ‘S’ to lowercase ‘s’